src/style.css

```diff
@@ -2,0 +1,109 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.search__container {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+.search__input {
+  color: #3d4e61;
+  width: 95%;
+  padding-left: 45px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300;
+  transition: border-color 0.3s;
+  box-shadow: 0 0 7px rgb(232, 230, 230);
+  font-family: inherit;
+  position: relative;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.search__input::placeholder {
+  color: #3d4e61;
+}
+
+.big {
+  margin-top: 20px;
+  height: 70px;
+  font-size: 16px;
+}
+
+.small {
+  margin-top: 20px;
+  height: 42px;
+  font-size: 14px;
+  box-shadow: 0 0 7px rgb(229, 228, 228);
+  text-shadow: 0 3px 4px #bbbec2;
```

Comment:
The '.small' class is adding both a margin-top and a box-shadow. Remember that it's recommended to use margins consistently (either top or bottom, but not both) and to not fix container sizes.